docs/large-bucket/DESIGN.md

```diff
@@ -184,7 +201,7 @@ type formatXLV2 struct {
 }
 ```
 
-## Limits
-
-- Minimum of 4 disks are needed for erasure coded configuration.
-- Maximum of 32 distinct nodes are supported in distributed configuration.
+## <a name="limits"></a>5. Limits
+Large-bucket support has the following limitations:
+* A minimum of 4 drives are needed for erasure-coded configurations.
+* A maximum of 32 distinct nodes are supported for distributed configurations.
```

Comment:
Should mention max 32 nodes per cluster, would be good to reference federation here for use cases that want to utilize more than 32 nodes or to add additional clusters.